### ISSUE TITLE
Add tahoe-lafs support

### ISSUE THREAD

**Issue by catskillmarina** [globaleaks/globaleaks-whistleblowing-software#880]
I suggest to add tahoe-lafs support in the backend. A layer of abstraction  in managing files would allow for swift and possibly other filesystems.

**Comment by evilaliv3** [comment]
@fpietrosanti  what do we want to do on this ticket given the e2e encryption support that we will end having with the OTF roadmap and the various discussion related to https://github.com/freedomofpress/securedrop/issues/106?

**Comment by fpietrosanti** [comment]
I don't think we need thaoe-lafs, for the same reasons discussed on SD ticket. If a volounteer want to abstract filesystem storage to work on different filesystem, let's see!
